project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.8.2
```

Comment:
Could you set this to `1.9.7` please in order to work around this problem: https://github.com/sbt/sbt/issues/7463